modules/heat_conduction/include/materials/GapConductanceConstant.h

```diff
@@ -0,0 +1,36 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef GAPCONDUCTANCECONSTANT_H
+#define GAPCONDUCTANCECONSTANT_H
+
+#include "Material.h"
+
+class GapConductanceConstant : public Material
+{
+public:
+  GapConductanceConstant(const InputParameters & parameters);
+
+  static InputParameters actionParameters();
+
+  virtual ~GapConductanceConstant() {}
```

Comment:
Note: You don't need to define an empty destructor in inherited classes where one of the bases has already declared a virtual destructor. You can simply remove this line or use the C++11 syntax to explicitly `delete` this destructor.

Reply:
Thanks for pointing that out.  I just pushed an updated version that removes that.